### PR TITLE
Create Skills and Bot: Minimized space needed for dropdown choices and improved (i) icon

### LINKED ISSUE #2741: Create Skills and Bot: Minimize space needed for dropdown choices and improve (i) icon
Category, language and skills name should require less space. In an earlier version of the page the form field names were listed above the field.

1. Please implement it in the same way again.
Something similar to this:
Screenshot from 2019-07-31 01-47-29

2. Regarding the i - icon
* Increase the size
* Make it blue
* For the tool tip and link displayed use the same colors as used on the top menu dropdown

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2741 

Changes: Create Skills and Bot: Minimized space needed for dropdown choices and improved (i) icon

Demo Link: https://pr-2765-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/cms/SkillCreator/SkillWizard.js [hunk 1]
```diff
@@ -326,9 +329,10 @@ const PreviewButton = styled.div`
 `;
 
 const DropDownWrap = styled.div`
-  display: flex;
```
Comment (fragm3) [inline src/components/cms/SkillCreator/SkillWizard.js]: Do not change this, we need flex

Comment (samagragupta) [reply src/components/cms/SkillCreator/SkillWizard.js]: @fragm3 Done

**Comment by samagragupta** [comment]
@akshatnitd Done the changes.

File: src/components/cms/SkillCreator/SkillWizard.js [hunk 2]
```diff
@@ -1120,43 +1125,51 @@ class SkillWizard extends Component {
...
                               {this.isBotBuilder
                                 ? 'Bot Name: '
                                 : 'Skill Name: '}
                             </DetailText>
-                            <NameField
+                            <OutlinedInput
                               margin="normal"
                               value={name}
                               onChange={this.handleExpertChange}
+                              variant="standard"
```
Comment (harsh-jindal) [inline src/components/cms/SkillCreator/SkillWizard.js]: Change this to `variant="outlined"`

For outlined select look at this : https://material-ui.com/components/selects/

**Comment by samagragupta** [comment]
@harsh-jindal Position of default image and upload icon are looking fine.
Capture

**Comment by samagragupta** [comment]
@harsh-jindal Made the changes

**Comment by samagragupta** [comment]
@akshatnitd Done the changes

**Comment by harsh-jindal** [comment]
@samagragupta Fix this.
Screenshot 2019-08-02 at 9 31 16 PM

**Pull request merged**